spec/integration/application/filebucket_spec.rb

```diff
@@ -70,10 +70,112 @@
         filebucket.command_line.args << 'fac251367c9e083c6b1f0f3181'
         filebucket.run
       }.to output(a_string_matching(
-        %r{Debug: HTTP GET https:\/\/127.0.0.1:#{port}\/puppet\/v3\/file_bucket_file\/md5\/fac251367c9e083c6b1f0f3181 returned 200 OK}
+        %r{Debug: HTTP GET https:\/\/127.0.0.1:#{port}\/puppet\/v3\/file_bucket_file\/md5\/fac251367c9e083c6b1f0f3181\?environment\=production returned 200 OK}
       ).and matching(
         %r{something to store}
        )).to_stdout
     end
   end
+
+  context 'diff' do
```

Comment:
I think diff needs to be skipped on windows?

Reply:
Ah yeah, can use `unless: Puppet::Util::Platform.windows? do` for that. Also might be good to add a context for `when diff is unavailable` and make sure the filebucket application prints a reasonable error message instead of exploding.